edenia/eden-spend-explorer
# fix(webapp): negative expense

### Negative expense

### What does this PR do?

- Resolve #400 
- Resolve #399 
- Resolve #395 

### Steps to test

1. Go to delegate view and search to mzdsisat1eos and validate that there aren't negative values
2. Open election 1, now you can see it.
